eclair-core/src/test/scala/fr/acinq/eclair/blockchain/watchdogs/BlockchainWatchdogSpec.scala

```diff
@@ -22,16 +22,19 @@ import com.typesafe.config.ConfigFactory
 import fr.acinq.bitcoin.Block
 import fr.acinq.eclair.TestTags
 import fr.acinq.eclair.blockchain.watchdogs.BlockchainWatchdog.{DangerousBlocksSkew, WrappedCurrentBlockCount}
+import fr.acinq.eclair.tor.Socks5ProxyParams
+import grizzled.slf4j.Logging
 import org.scalatest.funsuite.AnyFunSuiteLike
 
+import java.net.InetSocketAddress
 import scala.concurrent.duration.DurationInt
 
-class BlockchainWatchdogSpec extends ScalaTestWithActorTestKit(ConfigFactory.load("application")) with AnyFunSuiteLike {
+class BlockchainWatchdogSpec extends ScalaTestWithActorTestKit(ConfigFactory.load("application")) with AnyFunSuiteLike with Logging {
```

Comment:
I don't think you're using the `Logging` trait, it was probably for your early tests?